src/utils/parsers/common.ts

```diff
@@ -148,15 +148,22 @@ export abstract class BaseParser {
         
         for (const line of file.lines) {
             index += line.content.length - 1
+            let isComment = false
+            for (const commentChar of this.commentCharacters) {
```

Comment:
I think if we exclude it from the `fileContent` it will mess up the line numbers, e.g. excluding a single line comment will make the index of any variable used after be off by 1